WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/OrderSynchronizer.swift

```diff
@@ -23,9 +23,10 @@ struct OrderSyncProductInput {
     var id: Int64 = .zero
     let product: ProductType
     let quantity: Decimal
+    let discount: Decimal
```

Comment:
I saw this change after writing the previous comment: Would this change make it easier to declare for general scenarios and tests, where discounts are not involved? 
```suggestion
    var discount: Decimal = .zero
```